tests/projects/test_remote_pipeline.py

```diff
@@ -0,0 +1,134 @@
+import base64
+import json
+import unittest.mock
+
+import kfp.compiler
+import kubernetes
+import kubernetes.client
+
+import mlrun
+import tests.projects.base_pipeline
+
+
+class TestRemotePipeline(tests.projects.base_pipeline.TestPipeline):
+    def _set_functions(self):
+        self.project.set_function(
+            func=f"{self.assets_path}/remote_pipeline.py",
+            name="func1",
+            image="mlrun/mlrun",
+            handler="func1",
+        )
+        self.project.set_function(
+            func=f"{self.assets_path}/remote_pipeline.py",
+            name="func2",
+            image="mlrun/mlrun",
+            handler="func2",
+        )
```

Comment:
Both functions holds the same code, you don't really need 2 functions, just 1 function (with 2 handlers